src/util/feefrac.h

```diff
@@ -144,6 +187,39 @@ struct FeeFrac
         std::swap(a.fee, b.fee);
         std::swap(a.size, b.size);
     }
+
+    /** Compute the fee for a given size `at_size` using this object's feerate.
+     *
+     * This effectively corresponds to evaluating (this->fee * at_size) / this->size, with the
```

Comment:
Wouldn't it save some confusion if there was a type for `WU` and a type for vbytes?  If for example a `FeeFrac` is created as {sats /vbytes} then one wouldn't want to accidentally pass an argument calculated using a `wu` metric for `at_size` param (possibly).  In that case one would want to div by 4 first to get corresponding vbytes metric, however if there where strong types, then it could be converted automagically.